### ISSUE TITLE
Status Display - Not as expected

### ISSUE THREAD

**Issue by dangarfield** [petfriendamy/ff7-scarlet#12]
I don't think the status allocations are quite right, I've spotted it a few times,

- Example Enemy: Ghost (Formation 398)
- Status Immunities:
  - Scarlet
    - Silence, Stop, Frog, Darkness
  - Kujata (and also [Enemy Guide](https://github.com/Akari1982/q-gears_reverse/blob/master/ffvii/documents/final_fantasy_vii_enemy_mech.txt#L5875)
    - Silence, Stop, Frog, Paralysis

If you look in the kernel editor at many attacks that are 0xFF, you'll see that the 'paralysis' flag is often 'off' and I remember seeing a few statuses that we're quite what I thought they should be.

Are you following this:
https://wiki.ffrtt.ru/index.php/FF7/Battle/Status_Effects
It seems to produce results for me.

I'm not exactly sure how you read the status objects and map them, but I think the GUI here:
https://github.com/petfriendamy/ff7-scarlet/blob/e6cf116b5567d2e450ed861ece69d8ecbdeb5ff4/src/Shared/Controls/StatusesControl.Designer.cs#L75
shows paralysis in a different order. 

And
https://github.com/petfriendamy/ff7-scarlet/blob/e6cf116b5567d2e450ed861ece69d8ecbdeb5ff4/src/Shared/Controls/StatusesControl.cs#L45
appears to not have 'paralysis' included. 

I assume it is something to do with that.

Thanks!

**Comment by petfriendamy** [comment]
You're right, it looks like I forgot to include paralysis in the status list (also it was in the wrong order). Should be fixed in the latest canary. Feel free to test and let me know if there are any other issues!
